Rick&Morty/RickAndMorty/Scenes/Locations/Location List/LocationsListViewController.swift

```diff
@@ -57,12 +55,32 @@ private extension LocationsListViewController {
         tableView.dataSource = self
         tableView.delegate = self
     }
+    
+    func bindViewData() {
+        store
+            .$locations
+            .sink { [weak self] _ in
+                DispatchQueue.main.async {
```

Comment:
Maybe you don't need this here for now to avoid any unpleasant explanations

Reply:
I need it :) It does not work without it.... I will handle them 💪 😄